### PR TITLE
Add capitalize

### PR DESCRIPTION
**Pull request by alisavch**
new function capitalize released

### CODE REVIEW

File: strings.go [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
 		reverse = string(v) + reverse
 	}
 	return reverse
 }
+
+// Capitalize takes an input string and returns result that capitalizes the input string.
+func Capitalize(str string) string {
+	capitalize := strings.ToUpper(str)
+	return capitalize
+}
```
Comment (papahmsolo) [inline strings.go:19]: No need to declare a variable here

File: strings_test.go [hunk 2]
```diff
@@ -21,9 +20,32 @@ func TestReverse(t *testing.T) {
...
+		if got != v.out {
+			t.Errorf("Reverse(%q) = %q; want %q", v.in, got, v.out)
+		}
+	}
+}
+
+func TestCapitalize(t *testing.T) {
+	var capitalizeTests = []struct {
+		in, out string
+	}{
```
Comment (papahmsolo) [inline strings_test.go:33]: can we use short variable declaration (same for reverse)?

File: strings_test.go [hunk 3]
```diff
@@ -21,9 +20,32 @@ func TestReverse(t *testing.T) {
...
+		{"hello", "HELLO"},
+		{"привет", "ПРИВЕТ"},
+		{"你好", "你好"},
+		{"прывітанне", "ПРЫВІТАННЕ"},
+		{"cześć", "CZEŚĆ"},
+		{"Olá", "OLÁ"},
```
Comment (papahmsolo) [inline strings_test.go:39]: This basically test the same thing, try to test some different properties, say numbers instead

**Review by papahmsolo — CHANGES_REQUESTED** [review]
Looks good

Let's rename the module to match the repo location

File: strings.go (hunk 1 above)
Comment (alisavch) [reply strings.go]: ok thanks

File: strings_test.go (hunk 2 above)
Comment (alisavch) [reply strings_test.go]: yeah, of course

**Review by papahmsolo — APPROVED** [review]
lgtm 👍

**Pull request merged**